dev/tools/gen_defaults/data/list.json

```diff
@@ -57,7 +57,33 @@
   "md.comp.list.list-item.pressed.state-layer.color": "onSurface",
   "md.comp.list.list-item.pressed.state-layer.opacity": "md.sys.state.pressed.state-layer-opacity",
   "md.comp.list.list-item.pressed.trailing-icon.icon.color": "onSurfaceVariant",
+  "md.comp.list.list-item.selected.container.color": "secondaryContainer",
+  "md.comp.list.list-item.selected.dragged.label-text.color": "onSecondaryContainer",
+  "md.comp.list.list-item.selected.dragged.leading-icon.color": "onSurface",
+  "md.comp.list.list-item.selected.dragged.state-layer.color": "onSurface",
+  "md.comp.list.list-item.selected.dragged.state-layer.opacity": "md.sys.state.dragged.state-layer-opacity",
+  "md.comp.list.list-item.selected.dragged.trailing-icon.color": "onSurface",
+  "md.comp.list.list-item.selected.focus.label-text.color": "onSecondaryContainer",
+  "md.comp.list.list-item.selected.focus.leading-icon.color": "onSurface",
+  "md.comp.list.list-item.selected.focus.state-layer.color": "onSurface",
+  "md.comp.list.list-item.selected.focus.state-layer.opacity": "md.sys.state.focus.state-layer-opacity",
+  "md.comp.list.list-item.selected.focus.trailing-icon.color": "onSurface",
+  "md.comp.list.list-item.selected.hover.label-text.color": "onSecondaryContainer",
+  "md.comp.list.list-item.selected.hover.leading-icon.color": "onSurface",
```

Comment:
Lots of new tokens here. Maybe we should have an issue tracking widget update for this.

Reply:
As i understand these tokens aren't currently used by any widgets that support these tokens. We can keep a issue tracker to make sure to use new tokens.

Reply:
Thanks for helping review this PR! Your are right, these tokens should be used for the menu item in selected status and looks like we don't have this feature yet:) I just created an issue so we can collect new tokens to make sure M3 defaults are up-to-date: https://github.com/flutter/flutter/issues/153397